src/Symfony/Component/Form/FormBuilderInterface.php

```diff
@@ -32,8 +35,8 @@ public function add($child, string $type = null, array $options = []);
     /**
      * Creates a form builder.
      *
-     * @param string      $name The name of the form or the name of the property
-     * @param string|null $type The type of the form or null if name is a property
+     * @param string                               $name The name of the form or the name of the property
+     * @param class-string<FormTypeInterface>|null $type The type of the form or null if name is a property
```

Comment:
this is wrong. `class-string<FormTypeInterface>` does not mean "the name of a class extending FormTypeInterface`. It means "the name of FormTypeInterface"

Reply:
I think this is what does cover the correct meaning:

```
@template T of FormTypeInterface

@param class-string<T>|null $type
```